External/Plugins/HaXeContext/PluginMain.cs

```diff
@@ -166,6 +170,30 @@ public void HandleEvent(Object sender, NotifyEvent e, HandlingPriority priority)
                     ASCompletion.Context.ASContext.RegisterLanguage(contextInstance, "haxe");
                     CommandFactoryProvider.Register("haxe", new HaxeCommandFactory());
                     break;
+                case EventType.Trace:
+                    var libs = new List<string>();
+                    var length = TraceManager.TraceLog.Count - logCount;
+                    for (var i = 0; i < length; i++)
+                    {
+                        var item = TraceManager.TraceLog[logCount + i];
+                        var patterns = new []
+                        {
+                            "(Library )([A-Za-z_.0-9]+)( is not installed)",
+                            "(Could not find haxelib \")([A-Za-z_.0-9]+)(\", does it need to be installed?)"//openfl project
+                        };
+                        foreach (var pattern in patterns)
+                        {
+                            var match = Regex.Match(item.Message, pattern);
+                            if (match.Success) libs.Add(match.Groups[2].Value);
+                        }
+                    }
+                    logCount = TraceManager.TraceLog.Count;
+                    if (libs.Count == 0) return;
+                    var label = "Error";
+                    var text = $"Libraries {string.Join(", ", libs.ToArray())} is not installed\nDo you want to install their using haxelib?";
```

Comment:
Uhm... not sure...  printing all libraries may look really ugly in projects with tons of missing dependencies. Maybe I would use something like "Missing libraries detected. Would you like to try to install them through the package manager?" Also, I wouldn't use a `MessageBox` for this since it's modal and I think the user experience is not the best for this. The key could just be "Info.MissingLib" or similar.